privacyidea/lib/security/encryptkey.py

```diff
@@ -144,11 +152,15 @@ def __init__(self, config=None, logout=True):
         self.is_ready = False
 
         timeout = self.config.get("timeout") or DEFAULT_TIMEOUT
+        lock_dir = self.config.get("lock_dir") or DEFAULT_LOCK_DIR
```

Comment:
the keys in the config are `PI_HSM_MODULE_LOCK_DIR` and `PI_HSM_MODULE_TIMEOUT`.

Reply:
Thats right. This is the way, how it works.